src/amo/components/AddonReviewCard/index.js

```diff
@@ -493,6 +494,13 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
       >
         <div className="AddonReviewCard-container">
           {errorHandler.renderErrorIfPresent()}
+          {review && review.isDeleted && (
+            <Notice type="error" className="Addon-non-public-notice">
+              {i18n.gettext(
+                'This is not a public review. You are only seeing it because of elevated permissions.',
```

Comment:
The notice needs some margin below it to the spacing consistent. We use even increments like `2px`, `4px`, `6px`, `8px`, `12px`, `16px`. This is buried in a Sketch file somewhere. The best thing would be to match the spacing of surrounding elements but be sure to also match the spacing on smaller screens (second screenshot).

<img width="567" alt="screen shot 2019-02-22 at 10 18 16 am" src="https://user-images.githubusercontent.com/55398/53255732-d85f7780-368b-11e9-9793-fbe088cf6c20.png">
<img width="362" alt="screen shot 2019-02-22 at 10 18 41 am" src="https://user-images.githubusercontent.com/55398/53255733-d8f80e00-368b-11e9-99d6-478e48dbae87.png">